content/docs/command-reference/repro.md

```diff
@@ -18,15 +18,18 @@ positional arguments:
 
 ## Description
 
-`dvc repro` provides an way to regenerate data pipeline results, by restoring
-the dependency graph (a
-[DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) implicitly defined
-by the [stage files](/doc/command-reference/run) (DVC-files with dependencies)
-that are found in the <abbr>project</abbr>. The commands defined in these stages
-can then be executed in the correct order, reproducing pipeline results.
-
-> Pipeline stages are typically defined using the `dvc run` command, while
-> initial data dependencies can be registered by the `dvc add` command.
+`dvc repro` provides a way to regenerate data pipeline results, by restoring the
+dependency graph (a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph))
+implicitly defined by the [stages](/doc/command-reference/run) listed in the
+[`dvc.yaml` file](/doc/user-guide/dvc-files-and-directories#dvcyaml-file). The
+commands defined in these stages can then be executed in the correct order,
+reproducing pipeline results. `dvc repro` relies on the DAG definition that it
```

Comment:
last sentence is excessive .. repeats the first one

Reply:
Will be addressing these in #1615